main.tf

```diff
@@ -67,6 +67,14 @@ resource "azurerm_kubernetes_cluster" "main" {
     }
   }
 
+  role_based_access_control {
+    enabled = length(var.admin_group_object_ids) > 0
+    azure_active_directory {
```

Comment:
This block should be added with the `admin_group_object_ids` set

Reply:
@derkoe I'm not sure to understand what you mean.